### DISCUSSION TITLE
1️⃣ 1차 스터디를 위한 프로그래머스 Level 1 문제 선택

### DISCUSSION THREAD

**Discussion by hyeinisfree** [hyeinisfree/kings-algorithm-study#1] [문제 추천하기]
## 📚 1차 스터디를 위한 프로그래머스 Level 1 문제 선택
- 1차 스터디 기간 : 2022.02.07 ~ 2022.03.07
- week1, 2 : 프로그래머스 Level 1 문제 풀이
- 2022.02.02까지 스터디원 모두 Level 1 문제를 10문제씩 추천
- 코멘트 반응을 통해 최종 문제를 선택

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [로또의 최고 순위와 최저 순위](https://programmers.co.kr/learn/courses/30/lessons/77484)
- 추천인 : 김혜인

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [소수 만들기](https://programmers.co.kr/learn/courses/30/lessons/12977)
- 추천인 : 김혜인

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [K번째수](https://programmers.co.kr/learn/courses/30/lessons/42748)
- 추천인 : @hyeinisfree @sio2whocodes

**Reply by sio2whocodes** [reply]
추천

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [모의고사](https://programmers.co.kr/learn/courses/30/lessons/42840)
- 추천인 : @hyeinisfree @chaewss @sio2whocodes

**Reply by sio2whocodes** [reply]
추천

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [체육복](https://programmers.co.kr/learn/courses/30/lessons/42862)
- 추천인 : @hyeinisfree @chaewss @sio2whocodes

**Reply by sio2whocodes** [reply]
추천

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [예산](https://programmers.co.kr/learn/courses/30/lessons/12982)
- 추천인 : @hyeinisfree @chaewss

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [2016년](https://programmers.co.kr/learn/courses/30/lessons/12901)
- 추천인 : 김혜인

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [최소직사각형](https://programmers.co.kr/learn/courses/30/lessons/86491)
- 추천인 : @hyeinisfree @sio2whocodes @milsakim

**Reply by sio2whocodes** [reply]
추천

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [시저암호](https://programmers.co.kr/learn/courses/30/lessons/12926)
- 추천인 : 김혜인

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [이상한 문자 만들기](https://programmers.co.kr/learn/courses/30/lessons/12930)
- 추천인 : @hyeinisfree @chaewss

**Comment by chaewss** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [문자열 내 마음대로 정렬하기](https://programmers.co.kr/learn/courses/30/lessons/12915)
- 추천인 : 서채원

**Comment by chaewss** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [나머지가 1이 되는 수 찾기](https://programmers.co.kr/learn/courses/30/lessons/87389)
- 추천인 : 서채원

**Comment by chaewss** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [두 개 뽑아서 더하기](https://programmers.co.kr/learn/courses/30/lessons/68644)
- 추천인 : 서채원

**Comment by chaewss** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [약수의 합](https://programmers.co.kr/learn/courses/30/lessons/12928)
- 추천인 : 서채원

**Comment by chaewss** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [최대공약수와 최소공배수](https://programmers.co.kr/learn/courses/30/lessons/12940)
- 추천인 : 서채원

**Comment by chaewss** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [소수 찾기](https://programmers.co.kr/learn/courses/30/lessons/12921)
- 추천인 : 서채원

**Comment by sio2whocodes** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [행렬의 덧셈](https://programmers.co.kr/learn/courses/30/lessons/12950)
- 추천인 : 임수정

**Comment by sio2whocodes** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [콜라츠 추측](https://programmers.co.kr/learn/courses/30/lessons/12943)
- 추천인 : 임수정

**Comment by sio2whocodes** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [하샤드 수](https://programmers.co.kr/learn/courses/30/lessons/12947)
- 추천인 : 임수정

**Comment by sio2whocodes** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [나누어 떨어지는 숫자 배열](https://programmers.co.kr/learn/courses/30/lessons/12910)
- 추천인 : 임수정

**Comment by sio2whocodes** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [없는 숫자 더하기](https://programmers.co.kr/learn/courses/30/lessons/86051)
- 추천인 : @sio2whocodes @milsakim

**Comment by sio2whocodes** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [부족한 금액 계산하기](https://programmers.co.kr/learn/courses/30/lessons/82612)
- 추천인 : 임수정

**Comment by milsakim** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [두 개 뽑아서 더하기](https://programmers.co.kr/learn/courses/30/lessons/68644)
- 추천인 : @milsakim

**Comment by milsakim** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [3진법 뒤집기](https://programmers.co.kr/learn/courses/30/lessons/68935)
- 추천인 : 김혜지

**Comment by milsakim** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [약수의 개수와 덧셈](https://programmers.co.kr/learn/courses/30/lessons/77884)
- 추천인 : 김혜지

**Comment by milsakim** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [내적](https://programmers.co.kr/learn/courses/30/lessons/70128)
- 추천인 : 김혜지

**Comment by milsakim** [comment]
- 분류 : 프로그래머스 Level 1
- 문제 : [음양 더하기](https://programmers.co.kr/learn/courses/30/lessons/76501)
- 추천인 : 김혜지

**Comment by imim-seung** [comment]
분류 : 프로그래머스 Level1
문제 : 서울에서 김서방 찾기
추천인 : 김승미

**Comment by imim-seung** [comment]
분류 : 프로그래머스 Level1
문제 : 같은 숫자는 싫어
추천인 : 김승미
